docs/quickstarts/nextjs.mdx

```diff
@@ -98,15 +104,16 @@ The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides acti
 
   </Tab>
 
-  <Tab>
-    For this project, it makes sense to wrap the `<Component />` element with `<ClerkProvider>`. This makes the active session and user context accessible anywhere within `MyApp`.
 
-    ```tsx filename="_app.tsx" {2,6,8}
+  <Tab>
+    ```tsx filename="_app.tsx" {2,8-10}
       import '@/styles/globals.css'
       import { ClerkProvider } from "@clerk/nextjs";
       import type { AppProps } from "next/app";
       function MyApp({ Component, pageProps }: AppProps) {
         return (
+          // Active session and user context
+          // available to all children of MyApp

```

Comment:
same comments as [the one above](https://github.com/clerk/clerk-docs/pull/703/files#r1488356090)